tests/conftest.py

```diff
@@ -386,9 +409,12 @@ def create_lambda_function(
     memory_size,
     publish,
     region_name,
+    security_groups=None,
+    subnet_ids=None
 ):
     """Creates a Lambda function."""
     # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-locals
```

Comment:
```suggestion
    # pylint: disable=too-many-arguments,too-many-locals
```

You can disable multiple linters in one line. Reference:

- https://pylint.pycqa.org/en/latest/faq.html#how-to-disable-a-particular-message

Reply:
:+1

Reply:
Check this one again, please. It should be:

```python
# pylint: disable=too-many-arguments,too-many-locals
```

Reply:
okey even though the format I used looks to be also working, as the linter is passing, but if the documentation states the other format I'm changing it.